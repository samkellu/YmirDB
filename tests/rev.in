set a 1 2 3 4 5 6 7 8 9
get a
rev a
rev b
set b 123 22 a 44
reb b
list entries
bye
