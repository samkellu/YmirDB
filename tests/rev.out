> ok

> [1 2 3 4 5 6 7 8 9]

> ok

> no such entry

> ok

> > b [123 22 a 44]
a [9 8 7 6 5 4 3 2 1]

> bye
